canplayer.c

```diff
@@ -541,19 +547,25 @@ int main(int argc, char **argv)
 					break;
 				}
 
-				if (sscanf(buf, "(%llu.%llu) %s %s", &sec, &usec, device, afrbuf) != 4) {
+				if (sscanf(buf, "(%llu.%llu) %21s %6299s", &sec, &usec, device, afrbuf) != 4) {
```

Comment:
limiting the string reading is a good idea but is not in the scope of this patch.
Please create a separate patch with a separate commit message for it.

Reply:
Removed this, I can squash, or it can be squashed when merging ?